packages/quill/src/assets/_base.scss

```diff
@@ -0,0 +1,471 @@
+// Styles shared between snow and bubble
+
+$controlHeight: 24px !default;
+$inputPaddingWidth: 5px !default;
+$inputPaddingHeight: 3px !default;
+$colorItemMargin: 2px !default;
+$colorItemSize: 16px !default;
+$colorItemsPerRow: 7 !default;
+
+.ql-#{$themeName}.ql-toolbar,
+.ql-#{$themeName} .ql-toolbar {
+  &:after {
+    clear: both;
+    content: '';
+    display: table;
+  }
+
+  button {
+    background: none;
+    border: none;
+    cursor: pointer;
+    display: inline-block;
+    float: left;
+    height: $controlHeight;
+    padding: $inputPaddingHeight $inputPaddingWidth;
+    width: $controlHeight + ($inputPaddingWidth - $inputPaddingHeight) * 2;
+
+    svg {
+      float: left;
+      height: 100%;
+    }
+
+    &:active:hover {
+      outline: none;
+    }
+  }
+
+  input.ql-image[type='file'] {
+    display: none;
+  }
+
+  button:hover,
+  button:focus,
+  button.ql-active,
+  .ql-picker-label:hover,
+  .ql-picker-label.ql-active,
+  .ql-picker-item:hover,
+  .ql-picker-item.ql-selected {
+    color: var(--ql-active-color);
```

Comment:
We should not introduce CSS custom properties in this PR. We haven't decided the API yet so at least they should belong to a separate PR.

Reply:
Changed back to sass variables